src/components/ProfilePage/FavoriteHouses/FavoriteHouses.tsx

```diff
@@ -0,0 +1,39 @@
+import { Box, Button } from '@mui/material';
+import noPhoto from '../../../assets/images/nophoto.png';
+
+const FavoriteHouses = ({ favorites }: { favorites: BasicHouseData[] }) => {
+
+    const displayFavorites = (favorites: BasicHouseData[]) => {
+        if (favorites.length > 0) {
+            return (
+                favorites.map((favorite: BasicHouseData, i) => {
+                    console.log(favorite);
+                    return (
+                        <Box component="div" key={i.toString()}>
+                            <h4>
+                                {favorite.city}, {favorite.streetName} {favorite.streetNumber}
+                            </h4>
+                            <p >
+                                {favorite.price}zł/m<sup>2</sup>
+                            </p>
+                            <img src={favorite.photo_0 ? favorite.photo_0 : noPhoto} alt="House" />
+                            <p>{favorite.descriptionField}</p>
+                            <Button>more info</Button>
+                        </Box>
```

Comment:
Just a note to remember: This <Box /> is entirely copied from ListOfHouses, so during refactor we can make it a separate reusable component for both homepage and favorites and propably bought houses too